### ISSUE TITLE
Describe how to setup mscolab wsgi by gunicorn

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#494]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/494) by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

python-socketio / flask-socketio wrapper has to be added over Flask app to be imported from wsgi file

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Workaround is doing something like this and starting locally the server process

```
<Location /colab>
    ProxyPass  <http://localhost:8083/>
    ProxyPassReverse  <http://localhost:8083/>
```

‌

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

@{557058:3ffd4910-450c-422e-9ab9-726d7f2c7449} , are you taking this issue further?  I think the last error was in local setup?

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

I move this to 1.9.2 so that we can verify changes

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

apache2 seems not official supported.   
[https://github.com/miguelgrinberg/Flask-SocketIO/issues/1025](https://github.com/miguelgrinberg/Flask-SocketIO/issues/1025)  
  
we can use gunicorn and proxy it.   
  
a local copy of server.py 

  
gunicorn --bind 0.0.0.0:5000 --worker-class eventlet -w 1 server:app

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

This is more a documentation task. We need a description how to setup this by gunicorn, I rename it also

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

resolved by 58d3ea4
